### PR TITLE
Improved the Readme and add a Code of Conduct

### LINKED ISSUE #8194: Restructure the README file and add a Code of Conduct File
Since Anki-Android is now going to reach larger audiences who will be willing to contribute it seems to be essential that the README file has additional information about the tech stack used. Link to other related repositories such as the Anki Documentation one. Also similar to other projects we need a basic code of conduct file to refer to users in case there is some scuffle. This could be a good reference - https://github.com/facebook/jest#credits
Finally a section acknowledging contributors could be a huge morale booster some what like this - https://github.com/facebook/jest#credits

### PR DESCRIPTION
**Pull request by aflah02**
## Purpose / Description
To improve the Readme and add a Code of Conduct

## Fixes
Fixes #8194

## How Has This Been Tested?

No Code Changes

## Learning (optional, can help others)
Linked in the Issue Raised

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,17 +23,17 @@ A semi-official port of the open source [Anki](http://ankisrs.net/index.html) sp
...
-- supported contents: text, images, sounds, MathJax
-- add cards by intent from other applications like dictionaries
+- Night Mode
+- Whiteboard 
+- Progress Widget
+- Detailed Statistics
+- Syncing with AnkiWeb
+- Write Answers (optional)
+- Text-to-Speech integration
+-Mmore than 6000 Premade Decks
```
Comment (Arthur-Milchior) [inline README.md:33]: Typo with two M here

File: README.md [hunk 2]
```diff
@@ -23,17 +23,17 @@ A semi-official port of the open source [Anki](http://ankisrs.net/index.html) sp
...
-- detailed statistics
-- syncing with AnkiWeb
-- write answers (optional)
-- text-to-speech integration
-- more than 6000 premade decks
-- spaced repetition (supermemo 2 algorithm)
-- supported contents: text, images, sounds, MathJax
-- add cards by intent from other applications like dictionaries
+- Night Mode
+- Whiteboard
```
Comment (Arthur-Milchior) [inline README.md:27]: I'm not really used to the standards. Not sure whether caps are better here. I've little opininion

**Review by Arthur-Milchior — COMMENTED** [review]
Hi,

Thanks for contributing.
I'm going to have two level of answers.

I believe that this PR is too big. There are things that require a long discussion and things that are trivial. I believe that a PR should not have some part of the contribution that blocks other part. 

Also, commits should be descriptive. "Update" is really not descirptive. That means that it will be hard to read history and figure out what's going on and what a change was made. I'd like a commit to simply indicate "put list first letter in upper case in readme" and one "thanking open source collective contributors in readme", but not that many changes.

And then there is the big question of code of conduct. There is plenty of questions such as:
* do we want one
* do we need one
* which one to choose
* is it the right time to consider it

If we suddenly have a big explosion in contributors, it may be the right time, as the risk of needing one increase. However, this one is clearly not appropriate right now. For example "A working group of community members is committed to promptly addressing any reported issues" is utterly false. We can't just write this first and create the group later. In 2020, we were 20 contributors to code. I have spoken with half of them, most are not really active in the community.  You used pandas email address, maybe pandas is big enough to have people taking time on it. Seriously, we are having trouble just dealing with GSoC candidate, which are all very nice and contributing constructively. There is no way we get enough time to find a way to create a group and process to work on those issues or to form to really understand what exists elsewhere. 

I'll let the other mainteneurs comment if they want. But right now, I would not merge the PR. I'll repeat on the issue the relevant comments so that the discussion can exists if they want

**Comment by aflah02** [comment]
Oh understood @Arthur-Milchior I'll drop the idea for a Code of Conduct for now and raise small issues instead.
Thanks for all the suggestions

**Comment by david-allison-1** [comment]
I wouldn't merge this - for the time being I don't feel we need it and the time spent discussing would be better improving the app. 

Regardless, the decision of what the Code of Conduct is should come after consensus with the community.

File: README.md [hunk 3]
```diff
@@ -77,6 +77,15 @@ Contribute
...
 
+## Code of Conduct
+
+We take all Code of Conduct related queries very strictly. You can find the guidelines [here](https://github.com/ankidroid/Anki-Android/CODE_OF_CONDUCT.md).
+
+## Credits
+
+Thanks to all these wonderful people for contributing!
+[![](https://opencollective.com/html-react-parser/contributors.svg?width=890&button=false)](https://github.com/ankidroid/Anki-Android/graphs/contributors)
+
```
Comment (mikehardy) [inline README.md:88]: Strange, this does not look like I would expect. I like the jest example you linked, but this deviates.

They have this which results in different graphics showing up, that match my expectations a bit more closely

```

<a href="https://github.com/facebook/jest/graphs/contributors"><img src="https://opencollective.com/jest/contributors.svg?width=890&button=false" /></a>

### [Backers](https://opencollective.com/jest#backer)

Thank you to all our backers! 🙏

<a href="https://opencollective.com/jest#backers" target="_blank"><img src="https://opencollective.com/jest/backers.svg?width=890"></a>

### [Sponsors](https://opencollective.com/jest#sponsor)

Support this project by becoming a sponsor. Your logo will show up here with a link to your website.
```

File: README.md [hunk 4]
```diff
@@ -23,17 +23,17 @@ A semi-official port of the open source [Anki](http://ankisrs.net/index.html) sp
...
+- Whiteboard 
+- Progress Widget
+- Detailed Statistics
+- Syncing with AnkiWeb
+- Write Answers (optional)
+- Text-to-Speech integration
+-Mmore than 6000 Premade Decks
+- Spaced Repetition (Supermemo 2 Algorithm)
+- Supported Contents: Text, Images, Sounds, MathJax
+- Add Cards by Intent from other Applications like Dictionaries
```
Comment (mikehardy) [inline README.md:36]: Please remove all of these changes, I don't want to change these at the same time we add the contributors stuff

**Review by mikehardy — CHANGES_REQUESTED** [review]
Thanks for this - I like the idea as stated in the original issue, but then this PR went out of scope

As the others mentioned, go for a targeted change.

Let's focus on recognizing contributors to code, sponsors + contributors + backers from opencollective and leave it at that for now - one small set of changes that we can all agree on quickly

File: README.md (hunk 4 above)
Comment (aflah02) [reply README.md:36]: @mikehardy I'm sorry I forgot to close this I've made a new issue with linked changes as well - https://github.com/ankidroid/Anki-Android/issues/8201

**Pull request closed without merging**